genai/content.go

```diff
@@ -78,3 +78,21 @@ func ImageData(format string, data []byte) Blob {
 		Data:     data,
 	}
 }
+
+// Ptr returns a pointer to its argument.
```

Comment:
Maybe provide an example here in the doc of how this is useful for making pointers from constants?

Reply:
done